test/files/neg/t2102.check

```diff
@@ -1,6 +1,6 @@
 t2102.scala:2: error: type mismatch;
- found   : java.util.Iterator[Int]
- required: scala.collection.Iterator[_]
+ found   : Iterator[Int] (in java.util)
+ required: Iterator[_] (in scala.collection)
```

Comment:
Additional idea would be colour coding the different part of the path... No idea if this is something Scala team would want or not though. The aligning seems already very useful :)